src/mscorlib/shared/Interop/Windows/Kernel32/Interop.SetLastError.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        [DllImport(Libraries.Kernel32, SetLastError = true)]
+        internal static extern void SetLastError(int lastError);
```

Comment:
It is done here for CoreCLR: https://github.com/dotnet/coreclr/blob/a9b25d4aa22a1f4ad5f323f6c826e318f5a720fe/src/vm/dllimport.cpp#L917

And here for CoreRT: https://github.com/dotnet/corert/blob/7289a1e406acbb4265dc92587be83901f1716008/src/Common/src/TypeSystem/IL/Stubs/PInvokeILEmitter.cs#L97

@luqunl @yizhang82 If it is not done in MCG, it is a bug that should be fixed in MCG.